app/routes/oauth-stripe.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const {
+  Route
+} = Ember;
+
+export default Route.extend({
+  // model(params) {
```

Comment:
Will `code` and `state` be the only params passed here?  If so, we don't have to declare a hash with `code` and `state`:

```
model(params) {
  return this.get('store').createRecord('stripe-account', params);
}
```
Since params is already a hash for us.